sdk/core/azure-core/inc/azure/core/cryptography/key_encryption_key.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief A key which is used to encrypt, or wrap, another key.
+ */
+
+#pragma once
+
+#include <azure/core/context.hpp>
+
+#include <string>
+#include <vector>
+
+namespace Azure { namespace Core { namespace Cryptography {
```

Comment:
Given this is modifying core, I don't think we should ship this as part of the upcoming GA.